build.gradle

```diff
@@ -410,7 +410,7 @@ project('spring-rabbit') {
 		testImplementation 'io.micrometer:micrometer-tracing-test'
 		testImplementation 'io.micrometer:micrometer-tracing-integration-test'
 		testRuntimeOnly 'org.springframework:spring-web'
-		testRuntimeOnly "org.apache.httpcomponents:httpclient:$commonsHttpClientVersion"
+		testRuntimeOnly "org.apache.httpcomponents.client5:httpclient5:$commonsHttpClientVersion"
```

Comment:
Why doesn't `reactor-netty-http` works here for us?
I mean it is still OK since it is a test dep, but probably with Reactor in hands we may avoid and extra dep to manage.

Reply:
I will try with that.